cmd/exporter/main.go

```diff
@@ -146,6 +152,37 @@ type staticIssuer struct{ tok string }
 
 func (s staticIssuer) Issue(_ context.Context) (string, error) { return s.tok, nil }
 
+func selectTokenIssuer(flags Flags, ctrlClient client.Client, slurmClusterID types.NamespacedName, log logr.Logger) (interface {
```

Comment:
Since the `issuer` interface is being used in function return, moving it outside and reusing it would make this cleaner.

```suggestion
type issuer interface {
	Issue(ctx context.Context) (string, error)
}

func selectTokenIssuer(flags Flags, ctrlClient client.Client, slurmClusterID types.NamespacedName, log logr.Logger) issuer {
	var issuer issuer
	var standaloneIssuer *tokenstandalone.StandaloneTokenIssuer


```